packages/jest-runner/fuzz.ts

```diff
@@ -63,6 +63,18 @@ export function fuzz(
 		// affecting other tests, e.g. set a test specific timeout.
 		const localConfig = JSON.parse(JSON.stringify(fuzzingConfig));
 
+		if (process.env.JAZZER_LIST_FUZZTEST_NAMES != undefined) {
+			if (process.env.JAZZER_LIST_FUZZTEST_NAMES_PATTERN != undefined) {
+				if (
+					name.toString().match(process.env.JAZZER_LIST_FUZZTEST_NAMES_PATTERN)
```

Comment:
The check should probably include the whole "test path" including describe block. WDYT?

Reply:
I don't see a clear path for getting the describe blocks in the context we're in here … I _could_ go through all tests and see if there is one with the same name and collect the describe blocks on the way, but that would break pretty quickly as soon as two tests have the same name. Or is there some other context to get the context from?

Reply:
Nvm … Just found `currentTestState()`! :see_no_evil:

Reply:
Done!